Marlin/Marlin_main.cpp

```diff
@@ -1691,6 +1691,11 @@ static void do_blocking_move_to(float x, float y, float z, float feed_rate = 0.0
   feedrate = old_feedrate;
 }
 
+inline void do_blocking_move_to_axis(AxisEnum axis, float where, float feed_rate = 0.0) {
```

Comment:
All kinds of possibilities, to be sure! All I know is in the long run it's going to get less ugly. This is a lot like unraveling spaghetti.